ui/app/styles/components/search-select.scss

```diff
@@ -101,6 +101,13 @@ div > .ember-power-select-options {
   justify-content: space-between;
   border-bottom: $light-border;
 
+  .list-item-text {
+    max-width: 200px;
+    overflow-x: hidden;
+    text-overflow: ellipsis;
+    margin-right: 5px;
```

Comment:
probably only relevant for margin-right here, but could you use the `_size_variables.scss` vars here? Our goal, though we're far from reaching it, is to not have any hard coded size variables as one offs in scss files to help keep things consistent throughout the app. However, one quick search for 200px and you'll see we have not achieved it. So for the 200px, feel free to keep that one, but margin-right could be replaced by one of the spacing vars listed.

Reply:
Actually I think we can remove the definition here and use helpers in the application of this class below. Check out the comment on search-select.hbs.